cmake/GsTaichiTargets.cmake

```diff
@@ -51,42 +43,6 @@ if(_IMPORT_PREFIX STREQUAL "/")
   set(_IMPORT_PREFIX "")
 endif()
 
-# - [gstaichi_c_api] -------------------------------------------------------------
-
-add_library(gstaichi_c_api SHARED IMPORTED)
```

Comment:
note to reviewers: this is some extraneous c-api related stuff, that we didnt delete before, when we removed c api.